frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -34,26 +34,28 @@ We need the following collection metadata (i.e. details associated with your pub
   - Publication/preprint DOI: can be added later
   - URLs: any additional URLs for related data or resources, such as GEO or protocols.io - can be added later
 
-Datasets need the following information added to a single h5ad (AnnData 0.7) format file per CELLxGENE visualization:
+Datasets need the following information added to a single h5ad (AnnData 0.8) format file per CELLxGENE visualization:
```

Comment:
I would strike _per CELLxGENE visualization_ - the information is required by the schema for many feature requirements related to metadata ...

Reply:
@nayib-jose-gloria ? Pending editorial?

Reply:
https://github.com/chanzuckerberg/single-cell-data-portal/pull/3180/files#diff-3aab6608e02aeee4406a95606f0a84d5f03989f2ababdb4fb87416f1eb5f8c7e